admin/config.yml

```diff
@@ -131,6 +131,19 @@ collections:
             - { label: 'Column 2', name: 'col2', widget: 'markdown', required: false, i18n: true }
             - { label: 'Subheading 3', name: 'subheading3', widget: 'string', required: false, i18n: true}
             - { label: 'Column 3', name: 'col3', widget: 'markdown', required: false, i18n: true }
+      - label: 'Steps Page'
+        name: 'steps'
+        widget: 'object'
+        i18n: true
+        collapsed: true
+        required: false
+        fields:
+            - { label: 'Class', name: 'class', widget: 'string', required: false, i18n: duplicate }
+            - { label: 'Intro', name: intro, widget: 'markdown', required: false, i18n: true }
+            - { label: 'Step 1', name: step1, widget: markdown, required: false, i18n: true }
```

Comment:
I would have thought these values needed to be quotes since this is JSON mode, but it turns out not! 👍